variables.tf

```diff
@@ -77,84 +78,158 @@ variable "cluster_name" {
   description = "EKS Cluster Name"
 }
 
-variable "cluster_kms_key_arn" {
+variable "cluster_version" {
+  description = "Kubernetes `<major>.<minor>` version to use for the EKS cluster (i.e.: `1.21`)"
   type        = string
   default     = null
```

Comment:
We need to default to a version, no? That way users can be confident that we have tested against a particular version i.e. 1.21 in our case today. We may not always support the latest upstream version if user does not provide this input.

Reply:
This is the upstream module variable. I agree we should set the default to 1.21